nupic/research/frameworks/continuous_learning/cl_utils.py

```diff
@@ -0,0 +1,256 @@
+# ----------------------------------------------------------------------
+#  Numenta Platform for Intelligent Computing (NuPIC)
+#  Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+#  with Numenta, Inc., for a separate license for this software code, the
+#  following terms and conditions apply:
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero Public License version 3 as
+#  published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#  See the GNU Affero Public License for more details.
+#
+#  You should have received a copy of the GNU Affero Public License
+#  along with this program.  If not, see http://www.gnu.org/licenses.
+#
+#  http://numenta.org/licenses/
+#
+
+import sys
+import time
+
+import torch
+import torch.nn.functional as F
+from tqdm import tqdm
+
+
+def train_multi_model(
```

Comment:
This looks like a copy & paste from  `train_model` in `model_utils.y
Why not just add the `freeze_params` to the original `train_model` ?

Reply:
Just so people could use the original and not get errors. You recommend changing it?

Reply:
Set the `freeze_params` default value to `None` and check before using. We should try to follow the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle whenever possible